core/client/src/main/java/alluxio/client/block/StreamFactory.java

```diff
@@ -103,13 +103,14 @@ public static BlockInStream createRemoteBlockInStream(FileSystemContext context,
    * @param blockSize the block size
    * @param blockStart the start position of the block in the UFS file
    * @param address the worker network address
+   * @param mountId the id of the mount that this file is mapped to
    * @param options the in stream options
    * @return the input stream
    */
   public static BlockInStream createUfsBlockInStream(FileSystemContext context, String ufsPath,
-      long blockId, long blockSize, long blockStart, WorkerNetAddress address,
+      long blockId, long blockSize, long blockStart, WorkerNetAddress address, long mountId,
```

Comment:
I thought having `UFSUri` was the plan for a long time. Not sure though if we should target this in this PR. 
I am happy to refactor the code with ufsuri after this Alluxio-2743